policyengine_us/variables/gov/states/mo/tax/income/taxable_income/mo_taxable_income.py

```diff
@@ -45,5 +45,8 @@ def formula(person, period, parameters):
         unit_taxinc = max_(0, unit_mo_agi - unit_mo_deductions)
 
         # allocate unit taxable income by each individual's share of unit AGI
```

Comment:
```suggestion
        # Allocate tax unit taxable income by each individual's share of unit AGI.
        # Use a mask rather than where to avoid a divide-by-zero warning. Default to zero.
```